CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 2016-08-19 Release 0.1.0
```

Comment:
README could probably do with some sort of introductory paragraph before a 1.0.0?